src/applications/static-pages/mhv-signin-cta/sass/mhv-signin-cta.scss

```diff
@@ -0,0 +1,52 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+// Match the va-alert styling.
+.mhv-c-reg-alert {
```

Comment:
Note: This SASS is also a copy from `src/applications/mhv-landing-page/sass/mhv-landing-page.scss`. We need to refactor this file out to a common location and use it for our other alerts once this PR is approved.